ISISNeutronMuon/MDANSE
# Add new job which calculates the distinct part of the Van Hove function.

**Description of work**
The Van Hove function is related to the intermediate scattering function via a Fourier transform and the dynamic structure factor via a double Fourier transform.

This PR adds the distinct part of the Van Hove function and is spherically averaged so that it is a function of r=|$`\vec{R}`$| and normalized so that $`G(r, t) \rightarrow 1`$ as $`r \rightarrow \infty`$ for liquids or gases. Also, this will mean that $`G(r, 0) = \mathrm{PDF}(r)`$ and will be consistent with the $\mathrm{PDF}(r)$ calculated in MDANSE. Similarly to the PDF calculations, inter and intramolecular parts will be separated.

See Fig. 10 in  https://journals.aps.org/pre/abstract/10.1103/PhysRevE.51.4626 for an example of the distinct part of the Van Hove function in the literature. Here is an example calculated with the new job using the Ar36 trajectory.

![image](https://github.com/user-attachments/assets/3003d8d8-ef40-4af6-ae59-31a6d0d0a4a7)

We can see that the correlation hole from around $r=0$ to $3$ begins to fill up as time evolves. Eventually as  $`t \rightarrow \infty`$,  $`G(r, t) \rightarrow 1`$.

A job which calculates the self part of the Van Hove function and documentation will be added in later PRs.

**To test**
The cython distance histogram function was generalized for the Van Hove function calculations. Calculate the PDF on protos and save the results. Switch to this branch and rebuild MDANSE, calculate the PDF, and compare against the previous results, results should be the same. Run a Van Hove Function Distinct job and check the results, the G(r,t=0) result should be the same as PDF(r).
